test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```

Comment:
- [ ] Please follow the project requirement carefully, according to which your `test.md` file should be ignored by the `.gitignore` file. As I can see very well you added the `test.md` file into the `gitignore` but you did add the file after adding the `test.md` file on staging. So kindly remove the `test.md` file and make a commit and then try to add the file again and it will automatically be ignored by the *gitignore.*

### Project Requirements

![Screenshot from 2023-04-25 16-59-11](https://github.com/Arch-Noize/microverse-hello-world/assets/66476329/f9f93ae0-4be9-41b2-a014-c3970febbdbd)